### PR TITLE
Add doc on how to configure Kubelet with a file

### PR DESCRIPTION
**Pull request by mtaufen**
Follow up to https://github.com/kubernetes/kubernetes.github.io/pull/5523 that describes how to load Kubelet configuration from a file.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5694)

### CODE REVIEW

File: docs/tasks/administer-cluster/kubelet-config-file.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+{% capture steps %}
+
+## Create the config file
+
+The subset of the Kubelet's configuration that can be configured via a file
+is defined by the `KubeletConfiguration` struct
+[here (v1alpha1)](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/apis/kubeletconfig/v1alpha1/types.go).
+The configuration file must be a JSON or YAML representation of the parameters
+in this struct. Note that that this structure, and thus the config file API,
```
Comment (tengqm) [inline docs/tasks/administer-cluster/kubelet-config-file.md]: that that -> that

File: docs/tasks/administer-cluster/kubelet-config-file.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+is still considured alpha and is not subject to stability gurarantees.
```
Comment (tengqm) [inline docs/tasks/administer-cluster/kubelet-config-file.md]: considured -> considered

File: docs/tasks/administer-cluster/kubelet-config-file.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+For a trick to generate a configuration file from a live node, see
+[Reconfigure a Node's Kubelet in a Live Cluster](/docs/tasks/administer-cluster/reconfigure-kubelet).
+
+## Start a Kubelet process configured via the config file
+
+Start the Kubelet with the `KubeletConfigFile` feature gate enabled and the 
+Kubelet's `--init-config-dir` flag set to the location of the directory
+containing the `kubelet` file. The Kubelet will then load the parameters defined
+by `KubeletConfiguration` from the `kubelet` file, rather than from their 
+associated command-line flags.
```
Comment (tengqm) [inline docs/tasks/administer-cluster/kubelet-config-file.md]: so settings in the configuration file take precedence of the command line flags?

File: docs/tasks/administer-cluster/kubelet-config-file.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+{% endcapture %}
+
+{% capture discussion %}
+
+## Relationship to Dynamic Kubelet Config
+
+If you are using the [Dynamic Kubelet Configuration](https://github.com/kubernetes/features/issues/281)
```
Comment (tengqm) [inline docs/tasks/administer-cluster/kubelet-config-file.md]: I'd suggest we here use the link shown above [Reconfigure a Node's Kubelet in a Live Cluster](/docs/tasks/administer-cluster/reconfigure-kubelet). Referring to issues on github.com is not end user friendly.

File: docs/tasks/administer-cluster/kubelet-config-file.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+feature, the configuration provided via `--init-config-dir` will be considered
+the "last known good" configuration by the automatic rollback mechanism.
+
+Note that the layout of the files in the `--init-conifg-dir` mirrors the layout
```
Comment (tengqm) [inline docs/tasks/administer-cluster/kubelet-config-file.md]: typo: --init-config-dir

**Comment by steveperry-53** [comment]
@mtaufen Aside from typos mentioned by @tengqm, Docs LGTM.

**Comment by mtaufen** [comment]
Updated PTAL

**Comment by steveperry-53** [comment]
@mtaufen Docs LGTM. I'll merge this as soon as we get official tech approval from @tengqm.

**Comment by mtaufen** [comment]
SG, thanks Steve!

**Pull request merged**